spec/factories/sample_types.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
```

Comment:
not sure if i should delete these stub files ... we have many that have not been deleted.

Reply:
probably don't need if not being used yet